packages/app/src/modules/layout.js

```diff
@@ -24,6 +24,12 @@ export const ADD_TO_LAYOUT_OPTIONS = [
     { axisName: 'filters', name: i18n.t('Add to filter') },
 ];
 
+export const axisLabels = {
```

Comment:
Good idea to export this. Then you can also use it in DefaultAxis.js and YearOverYearAxis.js